src/_zkapauthorizer/_storage_server.py

```diff
@@ -173,6 +173,12 @@ class ZKAPAuthorizerStorageServer(Referenceable):
         default=attr.Factory(partial(namedAny, "twisted.internet.reactor")),
     )
 
+    def __attrs_post_init__(self):
+        # Avoid the default StorageServer ``allocate_buckets`` behavior of
+        # renewing leases on all existing shares in the same bucket.  It will
+        # still add leases to the newly uploaded shares.
+        self._original.set_implicit_bucket_lease_renewal(False)
```

Comment:
Per https://github.com/tahoe-lafs/tahoe-lafs/pull/1160#issuecomment-967446029, I don't think that setting this here makes sense (as opposed to passing in the value to the internal call).